### ISSUE TITLE
Feature Request: Add option to "enable like / dislike button" for embedded videos

### ISSUE THREAD

**Issue by random6969** [YePpHa/YouTubeCenter#1614]
I've noticed that on a lot of embedded videos, 
![like this one,](https://cloud.githubusercontent.com/assets/10361120/6975726/ab05b80a-d954-11e4-8bf9-764aa837a6ce.jpg)
there is no option to give the video a thumbs-up or thumbs-down. The only way to do so is to click the Youtube button to load the video on Youtube, and then rate it from there. It would greatly simplify things if the upvote / downvote functionality was added directly to embedded videos, perhaps in the upper right corner next to the "share" icon.

**Comment by q1k** [comment]
I think flash has those buttons  visible by default
for HTML player you can use the CSS style 
install stylish to use it

<code>
@namespace url(http://www.w3.org/1999/xhtml);
@-moz-document domain("youtube.com") {
.ytp-button-dislike,.ytp-button-like{
display: inline-block !important;
}
}
</code>

**Comment by Piter432** [comment]
Agreed. Sometimes it's frustating that I can't like/dislike YouTube video
placed in external site.

**Comment by random6969** [comment]
q1k, if it's not too much to ask, can you explain how to add that CSS style using the Stylebot extension in chrome? I tried adding it as 2 separate lines of code:
@namespace url(http://www.w3.org/1999/xhtml); 
@-moz-document domain("youtube.com") { .ytp-button-dislike,.ytp-button-like{ display: inline-block !important; } } 

but it gave me a syntax error.

**Comment by q1k** [comment]
What I gave above is the mozilla layout used by stylish. If you are using firefox, palemoon, seamonkey or any other similar browser, you would just copy paste the code from my other post and give the style a name.

For chrome you only need this code

<code>
.ytp-button-dislike,.ytp-button-like
{
display: inline-block !important;
}
</code>

![untitled](https://cloud.githubusercontent.com/assets/7927176/6993603/0155e3dc-dafa-11e4-9ba6-e0c885eb097b.png)

So on the left side you can name it as you like and make sure the enable box is checked,
then on the bottom select domain and enter 'youtube.com'.
In the code area enter the above code, this should make the like/dislike buttons always visible on the HTML embeded player.

In stylebot it should be pretty straightforward as well
enter the domain, enter the code, save.

**Comment by random6969** [comment]
Thanks q1k, your solution for chrome worked!

**Comment by random6969** [comment]
@q1k  I've noticed that while the like / dislike buttons are added to embedded videos, a few minutes after clicking one of them they reset to show neither button has been clicked (i.e.: I like a video at the 30 second mark, but after the 2:30 minutes mark the like is no longer shown on the video, both the like and dislike button appear to not have been clicked. In addition, going to the actual video on Youtube shows the original like from the embed, indicating it was carried out. This is only a minor annoyance, but I'd like to know if there's a way for the embedded video to actually remember the like / dislike I gave it.

**Comment by random6969** [comment]
@q1k If it's not too much trouble, can you explain what code to use for the new transparent youtube player? (https://www.youtube.com/testtube) The previous code you gave me for Stylish on chrome doesn't work with the new player.
